sdk/core/core-http/lib/policies/proxyPolicy.ts

```diff
@@ -26,21 +31,32 @@ function loadEnvironmentProxyValue(): string | undefined {
   return undefined;
 }
 
-export function getDefaultProxySettings(proxyUrl?: string): ProxySettings | undefined {
+export function getDefaultProxySettings(
+  proxyUrl?: string,
+  username?: string,
+  password?: string
```

Comment:
@HarshaNalluru With the new set of changes, we should revert the changes to this file

Reply:
In progress. Pushing the changes.